src/app/styles/components/_directions-form.scss

```diff
@@ -411,4 +405,60 @@
         }
 
     }
+
+    .directions-to {
+         &.isochrone-control {
+             display: none;
+         }
+
+         &.directions-text-input {
+             display: flex;
+         }
+     }
+
+     #isochrone-slider {
+        @include inputrange();
+        margin-right: 1rem;
+        width: 70%;
+    }
+
+    &.directions-destination-mode-within {
```

Comment:
I don't see this class anywhere, which seems related to the slider text looking rather un-styled
![image](https://cloud.githubusercontent.com/assets/6598836/22026382/9d1790e8-dc9e-11e6-855e-7d397530cefd.png)